src/cache/diagnostic_edmf_precomputed_quantities.jl

```diff
@@ -458,5 +458,60 @@ function set_diagnostic_edmf_precomputed_quantities!(Y, p, t)
 
     @. ᶜu⁰ = C123(Y.c.uₕ) + ᶜinterp(C123(ᶠu³⁰))
 
+    @. ᶜlinear_buoygrad = buoyancy_gradients(
```

Comment:
This function increases the allocation a bit. The allocation for `diagnostic_edmfx_aquaplanet` goes from 10704 to 11376 with this function. This is the same with prognostic edmfx I think, I just didn't try to fix it last time as the allocation only increases slightly, and the jet failure doesn't change. @charleskawczynski @nefrathenrici Anything that looks obvious to you?